### PR TITLE
Form as separate field containers

### LINKED ISSUE #59: Split form into separate components
The `/profile/edit` page is currently implemented as a single large form.

It would be more effective to break each input field down into a separate container:
- size of the form would be a lot smaller
- each form field container would then be reusable
- would be easier to test, as each component would have its own test

### PR DESCRIPTION
**Pull request by mcalthrop**
Closes #59.

### CODE REVIEW

File: src/shared/forms.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const messages = {
```
Comment (vincemtnz) [inline src/shared/forms.js:1]: Seems like this file is only used by `FormErrorMessages`

File: src/translations/en.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ export default {
   'aboutPage.title': 'About Us',
   'common.form.error.required': 'This field is required',
   'common.form.error.email': 'Email address is invalid',
+  'common.form.error.maxLength': 'May not be more than ' +
+  '{maxLength, plural, one {1 character} other {{maxLength} characters}} long',
```
Comment (vincemtnz) [inline src/translations/en.js]: Multiline strings are usually [preferred to concatenation](https://github.com/airbnb/javascript#6.4). I'm surprised our lint task did not catch this.

File: src/shared/forms.js (hunk 1 above)
Comment (mcalthrop) [inline src/shared/forms.js:1]: Yep, it is at this stage – but the nature of these messages is that they are could be used on more than one form, hence centralising them in `src/shared`.

File: src/translations/en.js (hunk 2 above)
Comment (mcalthrop) [inline src/translations/en.js]: Yeah, I hate concatenation. You mean using backticks?

**Pull request merged**